examples/js/loaders/GLTFLoader.js

```diff
@@ -2240,9 +2240,10 @@ THREE.GLTFLoader = ( function () {
 
 			}
 
-			// emissiveTexture and baseColorTexture use sRGB encoding.
+			// baseColorTexture, emissiveTexture, and specularGlossinessTexture use sRGB encoding.
```

Comment:
nit: specular is in sRGB but glossiness is in linear. Then the comment would be `specularTexture` instead of `specularGlossinessTexture`?